zork.js

```diff
@@ -0,0 +1,329 @@
+//globals
+const readline = require('readline');
+const readlineInterface = readline.createInterface(process.stdin, process.stdout);
+
+function ask(questionText) {
+    return new Promise((resolve, reject) => {
+        readlineInterface.question(questionText, resolve);
+    });
+}
+
+class Player{
+    constructor(name,title){
+        this.name = name,
+        this.title = title
+    }
+    inventory = [];
```

Comment:
Even though you're not passing any values in through the constructor you should still put all your properties inside the constructor and append them with the `this` keyword